cni/network/invoker_cns.go

```diff
@@ -179,14 +180,14 @@ func setHostOptions(nwCfg *cni.NetworkConfig, hostSubnetPrefix *net.IPNet, ncSub
 func (invoker *CNSIPAMInvoker) Delete(address *net.IPNet, nwCfg *cni.NetworkConfig, args *cniSkel.CmdArgs, options map[string]interface{}) error {
 
 	// Parse Pod arguments.
-	podInfo := cns.KubernetesPodInfo{PodName: invoker.podName, PodNamespace: invoker.podNamespace}
+	endpointId := GetEndpointID(args)
+	podInfo := cns.NewPodInfo(args.ContainerID, endpointId, invoker.podName, invoker.podNamespace)
 
 	orchestratorContext, err := json.Marshal(podInfo)
 	if err != nil {
 		return err
 	}
 
-	endpointId := GetEndpointID(args)
 	req := cns.IPConfigRequest{
 		OrchestratorContext: orchestratorContext,
```

Comment:
The orchestrator context was initillay podname + Podns, now it is the whole endpoint id, containerId and podname and podns....


I think in CNS we still treat orchestrator context to be podname+podns only